Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml

```diff
@@ -25,12 +28,21 @@ query: |
   | where Active == true
```

Comment:
This is an incorrect way to use the TI table. If you converted any in the past, you know it needs to have a summarize clause between lines 27 and 28. In this case I suggest you do add it.